include/ddwaf.h

```diff
@@ -161,6 +157,19 @@ struct _ddwaf_ruleset_info
     const char *version;
 };
 
+/**
+ * @ddwaf_metrics
+ *
+ * Structure containing total and per-rule runtimes
+ **/
+struct _ddwaf_metrics
+{
+    /** Total WAF runtime in microseconds **/
+    uint64_t total_runtime;
```

Comment:
Why not stick to one unit? 2^64-1 ns is like 585 years, so it's not like we need to reduce precision to get magnitude.

Reply:
Fair, I was also thinking about changing it.